backend/app/Services/Material/MaterialReclassificationService.php

```
@@ -0,0 +1,166 @@
+        User $by,
+        ?string $reason = null,
+    ): MaterialReclassification {
+        if (! in_array($toStatus, [MaterialLot::STATUS_RELEASED, MaterialLot::STATUS_QUARANTINE, MaterialLot::STATUS_REJECTED], true)) {
+            throw new \InvalidArgumentException("Unsupported target status: {$toStatus}.");
+        }
+
+        $fromStatus = $lot->status;
+
+        return DB::transaction(function () use ($lot, $toStatus, $by, $reason, $fromStatus) {
+            // Create the audit row first so the reject scrap movement can point at it.
+            $record = MaterialReclassification::create([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\MaterialReclassification::create().


(staticMethod.notFound)